wiki/News_Styling_Criteria/en.md

```diff
@@ -68,37 +74,35 @@ date: 2017-08-17 03:00:00 +0000
 
 ### Markdown
 
-The wiki uses a partial implementation of GFM (GitHub Flavored Markdown) with the following exceptions:
+The osu! wiki uses a partial implementation of GFM (GitHub Flavored Markdown) with the following exceptions:
 
 #### Titles
 
 News articles do not use traditional Markdown titles in any capacity whatsoever.
 
 #### Headers
 
-Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. Preferably, start with level 2 and use level 3 if the section belongs underneath the leading level 2 header. Never use headers to style or format text.
+Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. Never use headers to style or format text.
 
 #### Bold
 
 Bolding is reserved for highlighting objects or phrases of importance, be them players, scoring, or whatever else is contextually considered a focus of the article in question.
 
 #### Italics
 
-Italics is used uncommonly to accent phrases or statements of interest within the conversational register. It may also be used to designate names of other creative works, such as the _osu! Coffee Hour_ or the _Aspire Beatmapping Contests_.
+Italics is used uncommonly to accent phrases or statements of interest within the conversational register. It may also be used to designate names of other creative works, such as the *osu! Coffee Hour* or the *Aspire Beatmapping Contests*.
 
 #### Horizontal bar
 
 Horizontal bars must be used sparingly and only in situations where content needs to be cleanly and definitively separated from the rest of the article.
 
 #### Tables
 
-We do not recommend using Markdown tables for displaying data in news articles. Visualising the data using an image is likely to be more pleasing and more accessible to most users.
+Markdown tables can be used, but is not recommended for displaying data in news articles. Visualising the data using an image is likely to be more pleasing and more accessible to most users.
 
 ### Images
 
-#### Repository directory
-
-All images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the wiki shared image repository.
+All images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository.
 
 News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, with the exception of the ``.md`` suffix.
```

Comment:
do the double backticks serve any purpose here?

Reply:
Double grave marks are equivalent to single grave marks. @Ephemeralis was the one who put those there (as he was the one who initially wrote this document).